Source/Scene/ImplicitSubtree.js

```diff
@@ -108,19 +108,46 @@ Object.defineProperties(ImplicitSubtree.prototype, {
       return this._metadataExtension;
     },
   },
+
+  /**
+   * Gets the implicit tile coordinates for the root of the subtree.
+   *
+   * @type {ImplicitTileCoordinates}
+   * @readonly
+   * @private
+   */
+  implicitCoordinates: {
```

Comment:
This was useful to have for the more manual implicit tile coordinate processing that I'm doing